### ISSUE TITLE
Please add to the list of supported chips WBLC9 

### ISSUE THREAD

**Issue by esnet146** [kuba2k2/libretuya#86]
Successfully flashed and working
https://developer.tuya.com/en/docs/iot/wblc9-module-datasheet?id=K9hgglry2jp5h

Tested on device RGB+CW Lapm Tuya WBLC9 (BK7231T) + BP1658
https://www.elektroda.com/rtvforum/topic3925796.html

**Comment by kuba2k2** [comment]
WBLC9 is not a chip, it's a "board" - a small PCB with BK7231T *chip* on it. "Supported" in this context means more like "has board definition files" than "works". Any board having a supported chip (which you can find a list of [here](https://docs.libretuya.ml/docs/status/supported/#cpu-list)) will work.

**Comment by esnet146** [comment]
You are right, but there is a full description of the board. URL attached. or is something missing?

**Comment by kuba2k2** [comment]
Yes, the board documentation on docs.libretuya.ml, as well as pin definitions in code are missing.
